src/selectors/code.tsx

```diff
@@ -38,10 +38,16 @@ const getAssemblyViewNativeSymbolLib: Selector<Lib | null> = createSelector(
     if (profile === null || nativeSymbol === null) {
       return null;
     }
-    return profile.libs[nativeSymbol.libIndex];
+    return profile.libs[nativeSymbol.libIndex] ?? null;
```

Comment:
Wait, does this mean I missed something in the crash marker stack review and we forgot to populate profile.libs? Profiles with bad lib indexes in the nativeSymbols table are invalid.